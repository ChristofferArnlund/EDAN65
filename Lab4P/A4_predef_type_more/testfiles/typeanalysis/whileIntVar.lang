int m() {
	int k = 5;
	while(k) { //Not ok k is int!
		print(k);	
	}
}
