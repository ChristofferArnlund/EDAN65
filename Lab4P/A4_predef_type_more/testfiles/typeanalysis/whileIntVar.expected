Error at line 3: Expected type BOOL but found INT
